### PR TITLE
Added new device - inmp441max98357afastled

### PR DESCRIPTION
**Pull request by ProrokWielki**
* Updateed readme file
* Added dedicated getting stareted md file
* Implemented new Deveice
* Alligned all need files so the new divce can be selected and built.

### CODE REVIEW

File: PlatformIO/platformio.ini [hunk 1]
```diff
@@ -44,7 +44,7 @@ lib_deps =
 
 [env:esp32dev]
 
-;supported: M5ATOMECHO=0, MATRIXVOICE=1, AUDIOKIT=2, INMP441=3, INMP441MAX98357A=4
+;supported: M5ATOMECHO=0, MATRIXVOICE=1, AUDIOKIT=2, INMP441=3, INMP441MAX98357A=4, ESP32_POE_ISO=5, TAUDIO=6 INMP441MAX98357AFASTLED=7
```
Comment (Romkabouter) [inline PlatformIO/platformio.ini:47]: I think you missed a comma between the last two devices

File: PlatformIO/settings.ini.example [hunk 2]
```diff
@@ -2,7 +2,7 @@
 hostname=192.168.43.140
 deployhost=192.168.43.24
 siteId=satellite
-;supported: M5ATOMECHO=0, MATRIXVOICE=1, AUDIOKIT=2, INMP441=3, INMP441MAX98357A=4, ESP32_POE_ISO=5, TAUDIO=6
+;supported: M5ATOMECHO=0, MATRIXVOICE=1, AUDIOKIT=2, INMP441=3, INMP441MAX98357A=4, ESP32_POE_ISO=5, TAUDIO=6 INMP441MAX98357AFASTLED=7
 device_type=4
```
Comment (Romkabouter) [inline PlatformIO/settings.ini.example:6]: comma, I know both are comments but better to fix it straight away

**Comment by Romkabouter** [comment]
Thanks for the work! I have added some comments about a comma, other than that good work :)

File: PlatformIO/settings.ini.example (hunk 2 above)
Comment (ProrokWielki) [reply PlatformIO/settings.ini.example:6]: fixed

File: PlatformIO/platformio.ini (hunk 1 above)
Comment (ProrokWielki) [reply PlatformIO/platformio.ini]: fixed

**Comment by Romkabouter** [comment]
Thank you, I will soon update the version info and release a new version :)

**Pull request merged**